### ISSUE TITLE
ERR: Video has no edges!

### ISSUE THREAD

**Issue by draco1544** [Inrixia/Floatplane-Downloader#148]
I'am running the docker image and i have this message on every download : ERR: Video has no edges!

**Comment by draco1544** [comment]
It's seams to be an issue with the service and not the container

https://status.floatplane.com/

**Comment by Inrixia** [comment]
Yep, anytime this error shows up it means there are no servers a video can be downloaded from.

The same goes for ERRTIMEOUT most of the time as this indicates the server the downloader is trying to download from is not responding.

Edges being download servers.

Nothing we can do but wait for them to be restored. Thankfully you can check download status on the status site now as you linked.

**Comment by draco1544** [comment]
> Edges being download servers.

Yes, do you have a list of edges server ?

**Comment by Inrixia** [comment]
You can view all floatplane edges here. https://www.floatplane.com/api/v2/edges

Note this does not suggest that all of these are avalible for each video.

**Comment by Inrixia** [comment]
Closing this now as download servers will be back in the next few days. And this specific error is not going to show (at least for now) in the latest version.
